[Question]: 对于摘要的这个预训练大模型更改配置来替换模型时，会报出错误

### 请提出你的问题

对于摘要的这个预训练大模型 https://github.com/PaddlePaddle/PaddleNLP/tree/develop/applications/text_summarization/pretrain

文档说这里的默认model是可替换，但并没有指出如何替换。如果我需要将IDEA-CCNL/Randeng-Pegasus-523M-Summary-Chinese这个模型替换为另一个Randeng-Pegasus的模型（比如：PaddlePaddle/Randeng-Pegasus-523M-Summary-Chinese-SSTIA），更改配置后并不能正确加载模型，错误显示为：
```
AssertionError: The models name: PaddlePaddle/Randeng-Pegasus-523M-Summary-Chinese-SSTIA is not in task:[text_summarization]
```
当前的运行环境为：
pip list|grep  paddle
```
paddle-bfloat        0.1.2
paddle2onnx          1.0.0
paddlefsl            1.1.0
paddlehub            1.5.0
paddlenlp            2.5.1
paddlepaddle-gpu     2.3.0.post101
tb-paddle            0.3.6
```
请帮忙解决下，如何才能正确使用百度官方支持的其他模型来替换IDEA-CCNL/Randeng-Pegasus-523M-Summary-Chinese这个模型？谢谢

--- a/paddlenlp/taskflow/taskflow.py
+++ b/paddlenlp/taskflow/taskflow.py
@@ -255,8 +255,23 @@ TASKS = {
                 "task_flag": "text_summarization-IDEA-CCNL/Randeng-Pegasus523M-Summary-Chinese",
                 "task_priority_path": "IDEA-CCNL/Randeng-Pegasus-523M-Summary-Chinese",
             },
+            "IDEA-CCNL/Randeng-Pegasus-523M-Summary-Chinese-V1": {
+                "task_class": TextSummarizationTask,
+                "task_flag": "text_summarization-IDEA-CCNL/Randeng-Pegasus523M-Summary-Chinese-V1",
+                "task_priority_path": "IDEA-CCNL/Randeng-Pegasus-523M-Summary-Chinese-V1",
+            },
+            "PaddlePaddle/Randeng-Pegasus-238M-Summary-Chinese-SSTIA": {
+                "task_class": TextSummarizationTask,
+                "task_flag": "text_summarization-PaddlePaddle/Randeng-Pegasus-238M-Summary-Chinese-SSTIA",
+                "task_priority_path": "PaddlePaddle/Randeng-Pegasus-238M-Summary-Chinese-SSTIA",
+            },
+            "PaddlePaddle/Randeng-Pegasus-523M-Summary-Chinese-SSTIA": {
+                "task_class": TextSummarizationTask,
+                "task_flag": "text_summarization-PaddlePaddle/Randeng-Pegasus-523M-Summary-Chinese-SSTIA",
+                "task_priority_path": "PaddlePaddle/Randeng-Pegasus-523M-Summary-Chinese-SSTIA",
+            },
         },
-        "default": {"model": "IDEA-CCNL/Randeng-Pegasus-523M-Summary-Chinese"},
+        "default": {"model": "PaddlePaddle/Randeng-Pegasus-523M-Summary-Chinese-SSTIA"},
     },
     "word_segmentation": {
         "modes": {

